CourseApp/Program.cs

```diff
@@ -1,13 +1,75 @@
 ﻿namespace CourseApp
 {
     using System;
+    using System.Collections;
+    using static System.Math;
 
     public class Program
     {
         public static void Main(string[] args)
         {
-            Console.WriteLine($"Hello world");
+            var a = 2.0;
+            var b = 3.0;
+            ArrayList listTaskOne = new ArrayList() { 0.11, 0.36, 0.05 };
+            ArrayList listTaskTwo = new ArrayList() { 0.08, 0.026, 0.35, 0.41, 0.51 };
+            CalculateTasks(a, b, listTaskOne, listTaskTwo);
             Console.ReadLine();
         }
+
+        public static void FillTaskOneArrayList(double a, double b, ArrayList listTaskOne)
+        {
+            var xStart = (double)listTaskOne[0];
+            var xEnd = (double)listTaskOne[1];
+            var xDelta = (double)listTaskOne[2];
+            listTaskOne.Clear();
+            for (double x = xStart; x <= xEnd; x += xDelta)
+            {
+                listTaskOne.Add(x);
+            }
+
+            var listSize = listTaskOne.Count;
```

Comment:
Do we need to store count in variable?

Reply:
Yes, because the Array List grows like the count.